.github/workflows/macos.yml

```diff
@@ -37,12 +37,12 @@ jobs:
       - name: package
         run: |
           # 拷贝依赖
-          macdeployqt bin/release/${targetName}.app -qmldir=. -verbose=1 -dmg
+          macdeployqt bin/release/${env:targetName}.app -qmldir=. -verbose=1 -dmg
```

Comment:
By default, macOS uses bash as its shell, not powershell or pwsh ([Workflow syntax for GitHub Actions - GitHub Docs § Using a specific shell](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell)). In shells used by GitHub Actions other than powershell and pwsh, there is no such syntax as `${env:foo}` .